src/lib/rewriteReadmeToIncludeCardText.ts

```diff
@@ -26,7 +48,9 @@ const createCardText = ({
 
   const updateTime = showUpdateTime ? `  \nLast Updated on ${new Date().toLocaleString()}` : ''
 
-  return `<a href="https://lapras.com/public/${shareId}" target="_blank" rel="noopener noreferrer"><img src="${imageUrl}" width="${cardWidth}" ></a>${updateTime}`
+  const _alternativeText: string = getAlternativeText({ alternativeText, lang, shareId, score })
+
+  return `<a href="https://lapras.com/public/${shareId}" target="_blank" rel="noopener noreferrer"><img ${isCenter ? 'align="center"' : ''} alt="${_alternativeText}" src="${imageUrl}" width="${cardWidth}" ></a>${updateTime}`
```

Comment:
Thanks for adding a great option!

However, Even if I put align="center" on the img element, it is not centered. If you want to center it, you must align the parent block element.

```suggestion
  return `<p ${isCenter ? 'align="center"' : ''}><a href="https://lapras.com/public/${shareId}" target="_blank" rel="noopener noreferrer"><img alt="${_alternativeText}" src="${imageUrl}" width="${cardWidth}" ></a>${updateTime}</p>`
```